plugin.json

```diff
@@ -11,6 +12,7 @@
     }
   },
   "settings_schema": {
-      "header": "Configure this plugin directly in the config.json file. Learn more [in our documentation](https://github.com/mattermost/mattermost-plugin-welcomebot/blob/master/README.md).\n\nTo report an issue, make a suggestion or a contribution, [check the plugin repository](https://github.com/mattermost/mattermost-plugin-welcomebot)"
+      "header": "Configure this plugin directly in the config.json file. Learn more [in our documentation](https://github.com/mattermost/mattermost-plugin-welcomebot/blob/master/README.md).",
+      "footer": "To report an issue, make a suggestion or a contribution, [check the plugin repository](https://github.com/mattermost/mattermost-plugin-welcomebot)."
```

Comment:
I split this up to make it consistent with the other plugins. 1/5

Reply:
I would propose we keep it combined when there are no settings and the header/footer are short. It's more clear when there is one white box for it

![image](https://user-images.githubusercontent.com/13119842/58800546-27477200-85d6-11e9-8a6c-41ed8236b66a.png)

Reply:
Fine by me. Reverted.